hat/app/org/hatdex/hat/utils/Mailer.scala

```diff
@@ -200,9 +208,23 @@ class HatMailerImpl @Inject() (
       from = emailFrom,
       subject = messages("email.dataswift.auth.subject.verifyEmail"),
       bodyHtml = views.html.mails.emailAuthVerifyEmail(email, verificationLink).toString(),
-      bodyText = views.txt.mails
-        .emailAuthVerifyEmail(email, verificationLink)
-        .toString()
+      bodyText = views.txt.mails.emailAuthVerifyEmail(email, verificationLink).toString()
+    )
+    Done
```

Comment:
aha!! Do you think this might cause the emails to fail to be delivered? I have noticed that actually in some cases. I don't remember in which repo but in a controller, I was sending the email in the `yield` part of the for comprehension and I was never receiving the email. When I moved it inside the `for` then I was getting the email

Reply:
Good catch @xavji